UI Session/UI Session4/css/style.css

```diff
@@ -0,0 +1,280 @@
+}
+
+
+.logo, li {
+    font-weight: bold;
+}
+
+a {
+    display: inline-block;
+    text-decoration: none;
+    color: #fff;
+    font-size: 13px;
+    transition: 400ms;
+}
+
+a:hover {
+    color: #1B1464;
+    transform: scale(1.4, 1.4);
+}
+
+h1 {
+    color: #fff;
+    font-size: 2em;
+}
+
+.hero-center {
+    padding: 100px;
+    display: flex;
+    align-items: top;
+    justify-content: space-around;
+}
+
+.phone {
+    height: 450px;
+    transition: 400ms;
+}
+
+.phone:hover {
+    transform: scale(1.1, 1.1);
+}
+
+span {
+    display: block;
+}
+
+button {
+    margin: 10px;
+    padding: 10px 30px;
+    border-radius: 3px;
+    border: none;
+    color: #fff;
+    transition: 400ms;
+}
+
+button:hover {
+    transform: scale(1.2, 1.2);
+    box-shadow: 0px 2px 10px #fff;
+}
+
+button.left {
+    background-color: #F97F51;
+}
+
+button.right {
+    background-color: #575fcf;
+}
+
+main {
+    margin: 80px auto;
+    width: 80%;
+}
+
+main header {
+    text-align: center;
+    padding-bottom: 100px;
+}
+
+main header h2 {
+    color: #686de0;
+}
+
+main p {
+    font-size: 14px;
+    color: #95a5a6;
+}
+
+.perfectSectionMiddle {
+    display: flex;
+    margin: auto;
+    justify-items: center;
+    justify-content: space-evenly;
+    text-align: justify;
+}
+
+.descriptionSection {
+    flex-basis: 25%;
+    display: flex;
+    align-items: top;
+}
+
+.descriptionSection h2 {
+    font-size: 20px;
+}
+
+.slider {
+    display: block;
+    margin: 50px auto;
+    width: 50%;
+}
+
+.feature {
+    /* padding: 0; */
+    margin: 0;
+    color: #7f8c8d;
+}
+
+.col2 {
+    padding-left: 10px;
+}
+
+.perfectSectionMiddle i {
+    font-size: 40px;
+}
+
+.topScreenshot, .bottomScreenshot, .middleScreenshot {
+    display: flex;
+    padding: 50px;
+    align-items: center;
+    justify-content: space-around;
+}
+
+.screenshotDescription {
+    flex-basis: 30%;
+    text-align: justify;
+}
+
+.row1Reviews, .row2Reviews {
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.reviewSection blockquote {
+    flex-basis: 30%;
+}
+
+.appDownloadSection {
+    padding-top: 100px;
+    background-color: #9980FA;
+    display: flex;
+    justify-content: space-around;
+}
+
+.downloadSectionLeft {
+    flex-basis: 30%;
+}
+
+.downloadSectionLeft h2 {
+    font-size: 30px;
+    color: white;
+}
+
+.downloadSectionLeft p {
+    color: white;
+}
+
+.appDownloadSection img {
+    width: 400px;
+    flex-basis: 30%;
+}
+
+footer {
+    background-color: #40407a;
+    padding-bottom: 10px;
+}
+
+.topFooterSection {
+    width: 70%;
+    margin: auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.topFooterSection img {
+    display: block;
+    width: 100px;
+    height: 15px;
+
+}
+
+.topFooterSection a {
+    color: #9980FA;
+}
+
+footer > img {
+    display: block;
+    width: 80%;
+    height: 1px;
+    margin: auto;
+}
+
+.bottomFooterSection {
+    text-align: center;
+}
+
+.bottomFooterSection p {
```

Comment:
CSS class name should be like bottom-footer-section as it is more readable than camelcase.